lib/cartopy/mpl/ticker.py

```diff
@@ -34,46 +35,138 @@ class _PlateCarreeFormatter(Formatter):
     _target_projection = ccrs.PlateCarree()
 
     def __init__(self, degree_symbol=u'\u00B0', number_format='g',
-                 transform_precision=1e-8):
+                 transform_precision=1e-8, decimal=False,
+                 minute_symbol=u"'", second_symbol=u"''",
+                 seconds_number_format='g',
+                 auto_hide=True):
         """
         Base class for simpler implementation of specialised formatters
         for latitude and longitude axes.
 
         """
         self._degree_symbol = degree_symbol
-        self._number_format = number_format
+        self._degrees_number_format = number_format
         self._transform_precision = transform_precision
+        self._decimal = decimal
+        self._minute_symbol = minute_symbol
+        self._second_symbol = second_symbol
+        self._seconds_num_format = seconds_number_format
+        self._auto_hide = auto_hide
+        self._auto_hide_degrees = False
+        self._auto_hide_minutes = False
+        self._precision = 5  # locator precision
 
     def __call__(self, value, pos=None):
-        if not isinstance(self.axis.axes, GeoAxes):
-            raise TypeError("This formatter can only be "
-                            "used with cartopy axes.")
-        # We want to produce labels for values in the familiar Plate Carree
-        # projection, so convert the tick values from their own projection
-        # before formatting them.
-        source = self.axis.axes.projection
-        if not isinstance(source, (ccrs._RectangularProjection,
-                                   ccrs.Mercator)):
-            raise TypeError("This formatter cannot be used with "
-                            "non-rectangular projections.")
-        projected_value = self._apply_transform(value, self._target_projection,
-                                                source)
-        # Round the transformed value using a given precision for display
-        # purposes. Transforms can introduce minor rounding errors that make
-        # the tick values look bad, these need to be accounted for.
-        f = 1. / self._transform_precision
-        projected_value = round(f * projected_value) / f
+        if self.axis is not None:
+
+            if not isinstance(self.axis.axes, GeoAxes):
+                raise TypeError("This formatter can only be "
+                                "used with cartopy axes.")
```

Comment:
Can you please replace `cartopy axes` with `cartopy GeoAxes` for clarity.

Reply:
Fixed